index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>tribute page</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css" />
+</head>
+
+<body>
+    <div id="main">
+        <header id="title">
+            <h1>
+                Dr. Norman Borlaug
+            </h1>
+
+            <p>The man who saved a billion lives</p>
+        </header>
+        <div id="img-div">
+            <img src="dr-borlaug.jpg" alt="Dr.Borlaug" id="image" class="center">
+
+            <div id="img-caption">
+                <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields
+                    - part of his life-long war on hunger. </p>
+
+            </div>
+        </div>
+    </div>
+
```

Comment:
Please do not use spaces, or indentations without a reason. For readability, add two spaces of indentation.